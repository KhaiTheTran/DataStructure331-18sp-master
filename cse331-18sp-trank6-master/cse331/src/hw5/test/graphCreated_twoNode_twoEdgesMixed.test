# Test graphCreated_twoNode_twoEdgesMixed

# Create a graph
CreateGraph graphCreated_twoNode_twoEdgesMixed

# Add two nodes
AddNode graphCreated_twoNode_twoEdgesMixed xx
AddNode graphCreated_twoNode_twoEdgesMixed yy

# Add two edges
AddEdge graphCreated_twoNode_twoEdgesMixed xx yy xxyy
AddEdge graphCreated_twoNode_twoEdgesMixed yy xx yyxx

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
ListNodes graphCreated_twoNode_twoEdgesMixed
ListChildren graphCreated_twoNode_twoEdgesMixed xx
ListChildren graphCreated_twoNode_twoEdgesMixed yy