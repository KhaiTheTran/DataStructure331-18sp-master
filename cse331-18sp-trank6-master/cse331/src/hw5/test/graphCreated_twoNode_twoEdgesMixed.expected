# Test graphCreated_twoNode_twoEdgesMixed

# Create a graph
created graph graphCreated_twoNode_twoEdgesMixed

# Add two nodes
added node xx to graphCreated_twoNode_twoEdgesMixed
added node yy to graphCreated_twoNode_twoEdgesMixed

# Add two edges
added edge xxyy from xx to yy in graphCreated_twoNode_twoEdgesMixed
added edge yyxx from yy to xx in graphCreated_twoNode_twoEdgesMixed

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
graphCreated_twoNode_twoEdgesMixed contains: xx yy
the children of xx in graphCreated_twoNode_twoEdgesMixed are: yy(xxyy)
the children of yy in graphCreated_twoNode_twoEdgesMixed are: xx(yyxx)